.github/workflows/tsbs-test.yaml

```diff
@@ -0,0 +1,79 @@
+name: tsbs-test
+on:
+  pull_request:
+    branches:
+       - main
+jobs:
+  tsbs-test:
+    name: tsbs-test
+    env:
+      DB_URL: ${{ secrets.PERFORMANCE_DB }}
```

Comment:
It might be good to have a description of what this would contain, otherwise it will be hard for other to understand what is required here.